workflows/plugins/output_loaders/entities/consensus_genome_output.py

```diff
@@ -33,6 +33,7 @@ async def load(
     ) -> None:
         sars_cov_2 = raw_inputs.get("sars_cov_2") or raw_inputs.get("creation_source") == "SARS-CoV-2 Upload"
         wgs = entity_inputs.get("accession") is None  # if not sars_cov_2 and no accession is provided, it is a WGS run
+        print(raw_inputs.get("creation_source"))
```

Comment:
Do you still need this for debugging?

Reply:
nope, I think the reason that the earlier check wasn't working was because we aren't passing in "creation_source" to the workflow. It was printing as "None"

Reply:
Weird! It should be getting set when [we call the createWorkflowRun mutation from rails](https://github.com/chanzuckerberg/czid-web-private/blob/ac1909c2bd69d37ff3f54b453d8f41458ff50005/app/services/sample_entity_creation_service.rb#L235), I wonder why it's not going through 🤔

Reply:
oh weird!